_data/events1.yml

```diff
@@ -0,0 +1,78 @@
+- category: Online Event
+  category_name: Online Event
+  date: Tue, Aug 13, 2024
+  description: "Kedro is an open-source Python framework to create reproducible, maintainable,\
+    \ and modular data science code. It uses software engineering best practices to\
+    \ help you build production-ready data science pipelines.\nSchedule:\n\u23F0 19:00\

```

Comment:
Could you import the description before `Schedule`, because we have a limitation of characters right now to be shown here: 
![Uploading image.png…]()

Reply:
I can remove the 'Schedule' part. I only added it as the web page only gives the start of the session, not the end. The schedule had both. To get the session end time we would have to go that particular event's page.